lua/core/terminal.lua

```diff
@@ -85,7 +75,7 @@ end
 
 M._exec_toggle = function(exec)
   local binary = M._split(exec)[1]
-  if is_installed(binary) ~= true then
+  if vim.fn.executable(binary) ~= true then
```

Comment:
`vim.fn.executable` returns 1 on success not `true`
``` lua
print(1 == true) -> false
```

Reply:
oops.. thanks copy-paste!